docs/sms-send-how-to.md

```diff
@@ -3,20 +3,22 @@ title: Send SMS
 topic: sms
 type: how-to
 typeOrder: 1
-code: https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendSms.kt
+code: https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendTextSms.kt
 ---
 
 # How to send SMS
 
 ## Overview
 
-This example shows how you can send SMSes. In order to send SMSes your credentials need to have the
+This example shows how you can send text SMSes. In order to send SMSes your credentials need to have the
```

Comment:
Doesn't it show how to send "text and binary SMSes" ?

Reply:
Done